Makefile

```diff
@@ -8,7 +8,7 @@ run:
 	cargo run -- run --min-gas-prices 0uatom
 
 test:
-	cargo test --features=macros_test,it  --no-fail-fast 
+	RUST_BACKTRACE=1 cargo test --features=macros_test,it  --no-fail-fast 
```

Comment:
I don't think that it's necessary. You may replace it by environment from the command.

Reply:
True, but on the other hand if run test you want to see where error occurred without re-running them.